k8s/view-api/prod-view-api.yaml

```diff
@@ -25,50 +27,58 @@ spec:
       affinity:
         nodeAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
-          - weight: 10
-            preference:
-              matchExpressions:
-              - key: node-type
-                operator: In
-                values:
-                - high-memory
-          - weight: 1
-            preference:
-              matchExpressions:
-              - key: node-type
-                operator: In
-                values:
-                - general-purpose
+            - weight: 10
+              preference:
+                matchExpressions:
+                  - key: node-type
+                    operator: In
+                    values:
+                      - high-memory
+            - weight: 1
+              preference:
+                matchExpressions:
+                  - key: node-type
+                    operator: In
+                    values:
+                      - general-purpose
       containers:
         - name: view-api
           image: us.gcr.io/airqo-250220/airqo-view-api:latest
           imagePullPolicy: Always
           resources:
             requests:
-              # Average usage was 0.01 vCPU and 1400Mi over the last 30 days
+              # Average usage was 0.01 vCPU and 500Mi over the last 30 days
```

Comment:
interesting to learn that these comments in a config file do not bring about any rum time errors. @123MwanjeMike

Reply:
Not really @Baalmart 
Those are yaml style comments.
I leave them there to provide the reviewers of my code insights as to why I made the resource updates